openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -231,6 +233,7 @@ func (e *extension) cleanupOldResources(ctx context.Context, ns string) error {
 	if err != nil {
 		return err
 	}
+	// Leases - DomainMapping and CertManager related
```

Comment:
Nit: Maybe we could remove the comments as it is a straightforward and grouping is done per resource type not based on dm, certmanager etc.